apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2Test.java

```diff
@@ -699,11 +699,11 @@ static void assertPartAMatches(Beneficiary inBeneficiary, Coverage inCoverage) {
     currSegment = MedicareSegment.PART_A;
     assertNotNull(coverage);
     assertNotNull(beneficiary);
+    assertEquals("cancelled", coverage.getStatus().toCode());
 
     verifyCoverageClass("Part A");
     verifyMeta();
     verifyExtensionsPartA();
-    verifyCoverageStatus();
```

Comment:
why was this removed and the direct assertion added? do we still use this method in other places where we expect the status to be active? if so adding a method parameter to the method would be more consistent